src/App/App.less

```diff
@@ -1,15 +1,9 @@
-.forside {
+.Side {
   display: flex;
   flex-direction: column;
   background-color: #EFEFEF;
```

Comment:
NAV sine farger kan importeres fra en fil `_variabler.less` fra design biblioteket.
Sjekk hvordan vi har gjort det her: https://github.com/navikt/kontakt-oss/blob/master/src/Banner/Banner.less linje 1 og 5.
Du skal kunne "cmd-klikke" på `_variabler.less` for å se hvilke farger som kan brukes